pkg/generation/cnr.go

```diff
@@ -97,7 +97,10 @@ func GenerateCNR(nodeGroup atlassianv1.NodeGroup, nodes []string, name, namespac
 		}
 	}
 
-	return atlassianv1.CycleNodeRequest{
+    // Default/normalize priority for backward compatibility (not stored on CNR anymore)
+    _ = max(nodeGroup.Spec.Priority, 0)
```

Comment:
I don't follow what this is going

Reply:
Seems leftover, I agree this needs removal.